module/programs/helper.js

```diff
@@ -78,6 +78,11 @@ module.exports = class ProgramsHelper {
     return new Promise(async (resolve, reject) => {
 
       try {
+
+        if(!data.hasOwnProperty('endDate')){
+          data.startDate = new Date(data.startDate)
+          data.endDate = data.startDate.setFullYear( data.startDate.getFullYear(), data.startDate.getMonth() + 12 );
+        }
         
         let programData = {
```

Comment:
@ankitpws We can consider using something like - https://lodash.com/docs/4.17.15#assign to automatically add new keys everytime